spec/01-unit/08-router_spec.lua

```diff
@@ -5816,6 +5816,40 @@ do
       assert.truthy(match_t)
       assert.same(use_case[2].route, match_t.route)
     end)
+
+    it("works when route.priority is near 2^53 - 1", function()
```

Comment:
This test may not be adequate, I don't think it will fail even if `priority` inside the test is set higher than `2^53 - 1`. What we need is to make sure after inserting a route, `GET /routes/uuid` can return a integer value of `2^53 - 2`, otherwise we will confuse a lot of systems like decK.

Reply:
Basically, we are making sure `priority` can be set and retrieved without losing precision.

Reply:
done.